rss2email/config.py

```diff
@@ -70,6 +71,8 @@ def setup_html2text(self, section='DEFAULT'):
         ### Addressing
         # The email address messages are from by default
         ('from', 'user@rss2email.invalid'),
+        # The User-Agent default string (__VERSION__ is replaced)
+        ('user-agent', 'rss2email/__VERSION__ ({})'.format(__url__)),
```

Comment:
Would it be possible to interpolate `__version__` in the same way as you're interpolating `__url__` here?